pkg/client/core/api/rest/cluster_rest.go

```diff
@@ -21,6 +24,11 @@ func (a *clusterAPI) DeleteCluster(opts api.ClusterDeleteOpts) error {
 	return a.client.DoDelete(ClusterTarget, &opts)
 }
 
+func (a *clusterAPI) GetClusterSecurityGroupID(opts api.ClusterSecurityGroupIDGetOpts) (*api.ClusterSecurityGroupID, error) {
+	id := &api.ClusterSecurityGroupID{}
+	return id, a.client.DoGet(ClusterSecurityGroupIDTarget, &opts, &id)
```

Comment:
Pointer to a pointer? I don't think thats necessary

Reply:
* Insert xzibit here *

Not sure why it worked though. Fixed anyway.